packages/core/src/tracing/instrumentation/frameworks/fastify.js

```diff
@@ -97,5 +97,5 @@ function annotateHttpEntrySpanWithPathTemplate(app, opts) {
     return;
   }
 
-  span.data.http.path_tpl = (app.basePath || '') + (opts.url || opts.path || '/');
+  span.data.http.path_tpl = (app.prefix || '') + (opts.url || opts.path || '/');
```

Comment:
We still need to support older versions of fastify, too. I guess we could check for the existence of `app.prefix` and use that if it exists, otherwise fall back to `app.basePath`?

Reply:
prefix and basePath do both exist but where changed to getters:
https://github.com/fastify/fastify/blob/7eaaabcbc376d3fdf6e14da146ffb3f5f301a7a3/fastify.js#L187

We have to check if basePath is a getter (which emits the warning). If it is, we have to use the prefix getter.